contracts/mocks/SimpleServiceManager.sol

```diff
@@ -95,4 +95,10 @@ contract SimpleServiceManager is IWavsServiceManager {
     function getLastCheckpointTotalWeight() external view returns (uint256) {
         return lastCheckpointTotalWeight;
     }
+
+    function getLatestOperatorForSigningKey(
+        address signingKey
+    ) external pure override returns (address) {
+        return signingKey;
```

Comment:
Don’t you want to return operator?
Or this is just a simple mock that returns identity?

Reply:
Yep just simple mock for wavs tests